src/zc/buildout/testing.py

```diff
@@ -56,6 +61,17 @@ def cat(dir, *names):
     with open(path) as f:
         print_(f.read(), end='')
 
+def eqs(a, *b):
+    a = set(a); b = set(b)
+    return None if a == b else (a - b, b - a)
+
+def clear_here():
+    for name in os.listdir('.'):
+        if os.path.isfile(name):
```

Comment:
I suggest adding `or os.path.islink(name)`, otherwise you can get an `OSError: Cannot call rmtree on a symbolic link` in the else branch.  (Which is much better than my $HOME getting accidentally erased, so kudos to Python stdlib developers!)